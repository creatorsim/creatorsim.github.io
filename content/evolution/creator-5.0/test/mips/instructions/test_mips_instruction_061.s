
#
# Creator (https://creatorsim.github.io/creator/)
#

    
.text
main:
    li.d $FP0, 256.0
  	li.d $FP2, -3481.0
    li.d $FP4, 5476.0
    
    li.s $f6, -9801.0
    li.s $f7, 8281.0
    li.s $f8, -9216.0
    
    abs.d $FP10, $FP0
    abs.d $FP12, $FP2
    abs.d $FP14, $FP4
    
    abs.s $f16, $f6
    abs.s $f17, $f7
    abs.s $f18, $f8
    
    li $v0, 10
    syscall

