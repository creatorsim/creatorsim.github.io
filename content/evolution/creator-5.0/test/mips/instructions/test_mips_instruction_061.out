
cr[PC]:0x8c; ir[1,at]:0xc6100000; ir[2,v0]:0xa; sfpr[f0]:0x40700000; sfpr[f2]:0xC0AB3200; sfpr[f4]:0x40B56400; sfpr[f6]:0xC6192400; sfpr[f7]:0x46016400; sfpr[f8]:0xC6100000; sfpr[f10]:0x40700000; sfpr[f12]:0x40AB3200; sfpr[f14]:0x40B56400; sfpr[f16]:0x46192400; sfpr[f17]:0x46016400; sfpr[f18]:0x46100000; dfpr[FP0]:0x4070000000000000; dfpr[FP2]:0xC0AB320000000000; dfpr[FP4]:0x40B5640000000000; dfpr[FP6]:0xC619240046016400; dfpr[FP8]:0xC610000000000000; dfpr[FP10]:0x4070000000000000; dfpr[FP12]:0x40AB320000000000; dfpr[FP14]:0x40B5640000000000; dfpr[FP16]:0x4619240046016400; dfpr[FP18]:0x4610000000000000; keyboard[0x0]:''; display[0x0]:''; 

